index.html

```diff
@@ -1,18 +1,22 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <script src="js/tiles.js"></script>
-    <script src="js/levels.js"></script>
-    <script src="js/player.js"></script>
-    <script src="js/gui.js"></script>
-    <script src="js/level.js"></script>
-    <!--<meta http-equiv="refresh" content="5">-->
-    <link rel="stylesheet" href="css/tiles.css">
-    <title></title>
-  </head>
-  <body>
-    <div id="tiles">
-    </div>
-  </body>
+<head>
```

Comment:
An HTML element's start tag may be omitted if the first thing inside the HTML element is not a comment.

sometimes omitting elements may lead to different renders on different browsers

Reply:
Please add `<html>`

Reply:
I have <html> in the start. Please can you specify where to add ?